xbmc/music/infoscanner/MusicInfoScanner.h

```diff
@@ -188,7 +189,7 @@ class CMusicInfoScanner : CThread, public IRunnable
   int GetPathHash(const CFileItemList &items, std::string &hash);
   void GetAlbumArtwork(long id, const CAlbum &artist);
 
-  bool DoScan(const std::string& strDirectory);
+  virtual bool DoScan(const std::string& strDirectory);
```

Comment:
I'm no expert on this, but should this really be virtual here? I would have thought we should loose the virtual and add an ```override```? But I may be out of my waters here.